packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -121,23 +124,19 @@ contract SortedOracles is ISortedOracles, Ownable, Initializable {
   /**
    * @notice Updates an oracle value and the median.
    * @param token The address of the token for which the Celo Gold exchange rate is being reported.
-   * @param numerator The amount of tokens equal to `denominator` Celo Gold.
-   * @param denominator The amount of Celo Gold equal to `numerator` tokens.
+   * @param value The amount of tokens equal to Celo Gold represented as a FixidityLib fraction
    * @param lesserKey The element which should be just left of the new oracle value.
    * @param greaterKey The element which should be just right of the new oracle value.
    * @dev Note that only one of `lesserKey` or `greaterKey` needs to be correct to reduce friction.
    */
-  function report(
-    address token,
-    uint256 numerator,
-    uint256 denominator,
-    address lesserKey,
-    address greaterKey
-  ) external onlyOracle(token) {
+  function report(address token, uint256 value, address lesserKey, address greaterKey)
+    external
+    onlyOracle(token)
+  {
     uint256 originalMedian = rates[token].getMedianValue();
-    uint256 value = numerator.mul(DENOMINATOR).div(denominator);
+    FixidityLib.Fraction memory fractionalValueReported = FixidityLib.wrap(value);
     if (rates[token].contains(msg.sender)) {
-      rates[token].update(msg.sender, value, lesserKey, greaterKey);
+      rates[token].update(msg.sender, fractionalValueReported.unwrap(), lesserKey, greaterKey);
```

Comment:
Seems we're changing the type of values stored in `rates` to unwrapped Fixidity fractions?   Then we should note in the PR that the changes are not backwards compatible.

Reply:
I have added a comment in the PR reg. that